packages/replay-debugger-ui/src/components/replay-debugger/replay-debugger.tsx

```diff
@@ -51,62 +56,12 @@ const ReplayDebuggerWrapped: FunctionComponent = () => {
               "sm:p-6"
             )}
           >
-            <button
-              type="button"
-              className={cx(
-                "flex-grow",
-                "inline-flex",
-                "items-center",
-                "justify-center",
-                "px-4",
-                "py-2",
-                "border",
-                "border-transparent",
-                "text-sm",
-                "font-medium",
-                "rounded-md",
-                "shadow-sm",
-                "text-white",
-                "bg-violet-600",
-                "hover:bg-violet-700",
-                "focus:outline-none",
-                "focus:ring-2",
-                "focus:ring-offset-2",
-                "focus:ring-violet-500"
-              )}
-              onClick={onCheckNextEventTarget}
-              disabled={loading}
-            >
-              Check next event target
-            </button>
-            <button
-              type="button"
-              className={cx(
-                "flex-grow",
-                "inline-flex",
-                "items-center",
-                "justify-center",
-                "px-4",
-                "py-2",
-                "border",
-                "border-transparent",
-                "text-sm",
-                "font-medium",
-                "rounded-md",
-                "shadow-sm",
-                "text-white",
-                "bg-violet-600",
-                "hover:bg-violet-700",
-                "focus:outline-none",
-                "focus:ring-2",
-                "focus:ring-offset-2",
-                "focus:ring-violet-500"
```

Comment:
I find it hard to quickly skim to get the core content/structure of the code when long lists of classes mixed in with functional structure & behaviour, or tell that these buttons/classes are identical at a glance, so pulled out VioletButton below.